benchmarks/profiler/profile_sla.py

```
@@ -132,7 +143,7 @@ async def run_profile(args):
                 raise ValueError(
                     "Must provide --aic-model-name when using --use-ai-configurator."
                 )
-            if not args.backend_version:
+            if not args.aic_backend_version:
                 raise ValueError(
                     "Must provide --backend-version when using --use-ai-configurator."
                 )
```

Ruff (0.13.3)

Abstract `raise` to an inner function

(TRY301)